tools/cmake/mbed_greentea.cmake

```diff
@@ -14,8 +14,12 @@ set(MBED_ROOT ${CMAKE_CURRENT_LIST_DIR}/../.. CACHE INTERNAL "")
 # TEST_INCLUDE_DIRS - Test suite include directories for the test
```

Comment:
In commit message for "CMake: tests: Support skipping unsupported test with reason ":

```diff
- to indicate a test is skip
+ to indicate a test is skipped
```

Reply:
Good catch. Fixed now.